x-pack/plugins/maps/public/connected_components/toolbar_overlay/toolbar_overlay.test.tsx

```diff
@@ -14,6 +14,13 @@ jest.mock('../../kibana_services', () => {
     getMapsCapabilities() {
       return { save: true };
     },
+    getEMSSettings() {
```

Comment:
nit: I wonder if it's possible to have a common / default mock of the kibana_services so that you don't need to update so many files every time something new needs to be added.

Reply:
Definitely not something to change in this PR (or even very soon), but we usually do it in Dashboard by providing a sensible default for the mock in `jest_setup` [example](https://github.com/elastic/kibana/blob/main/src/plugins/dashboard/jest_setup.ts). Then for any test that needs a variable injected we override the service with a jest.fn() [example](https://github.com/elastic/kibana/blob/f4e79d69b49d606f84b34f4998a048535db411b0/src/plugins/dashboard/public/dashboard_actions/clone_panel_action.test.tsx#L42).

It works pretty well, but I'm not sure how easily that would be applicable to the maps service infrastructure.